src/main/java/org/spongepowered/common/mixin/core/block/MixinBlock.java

```diff
@@ -179,4 +180,12 @@ private static void onSpawnAsEntity(net.minecraft.world.World worldIn, BlockPos
             return;
         }
     }
+
+    @Override
+    public String toString() {
+        return SpongeCatalogType.toStringHelperTranslatable(this)
```

Comment:
If you're going to change what `Block.toString()` returns, at least store this in a field so it's not re-generated every single time.